Source/WTF/wtf/text/StringView.h

```diff
@@ -586,18 +586,17 @@ inline StringView StringView::substring(unsigned start, unsigned length) const
 
 inline UChar StringView::characterAt(unsigned index) const
 {
-    RELEASE_ASSERT(index < length());
     if (is8Bit())
-        return characters8()[index];
-    return characters16()[index];
+        return span8()[index];
+    return span16()[index];
 }
 
 inline UChar StringView::unsafeCharacterAt(unsigned index) const
 {
     ASSERT(index < length());
     if (is8Bit())
-        return characters8()[index];
-    return characters16()[index];
+        return span8().data()[index];
+    return span16().data()[index];
```

Comment:
We should consider getting rid of `unsafeCharacterAt` unless it’s a measurable performance win, no?

Reply:
It is private and only used from iterator code so it's not terrible. But yes, we should try and get rid of it if there is no performance impact. I'll do separately though as this requires further perf testing.